app/views/shared/_booking_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= simple_form_for @service.bookings.build, url: service_bookings_path(@service), method: :post, wrapper: :vertical_form do |form| %>
+  <%= form.input :name, placeholder: 'Name' %>
+  <%= form.input :email, placeholder: 'Email' %>
+  <%= form.input :email_confirmation, placeholder: 'Confirm Email' %>
+  <%= form.input :phone, placeholder: 'Phone' %>
```

Comment:
Did we create a migration column for phone numbers in one of the models?

Reply:
good catch, this column has to be added the the User model. Will have it done after this is done in a new branch